installer/datafiles/ruby.data

```diff
@@ -3,38 +3,14 @@ RUBY_DEST:                '/opt/microsoft/omsagent/ruby'
 
 %Files
```

Comment:
Hi Narine,

What you said is not correct. The new ruby.data file that you generated absolutely has changes, and has changes that you need. For example, one of the changes is:

${{RUBY_DEST}}/lib/ruby/gems/2.2.0/doc/mysql2-0.4.4/ri/ext/mysql2/page-Makefile.ri; ${{RUBY_INT}}/lib/ruby/gems/2.2.0/doc/mysql2-0.4.4/ri/ext/mysql2/page-Makefile.ri; 644; root; root

Either check in the changes to genRubyInstaller.sh first (along with documentation on how to update Ruby), or include those files in this pull request. I don't care which. But if you needed to update genRubyInstaller.sh as part of this work, then that should either be checked in as part of this work, or checked in first so it's clearer how this work was done.

I'm perfectly fine helping your team, but it's super important to me that we improve documentation to ease my workload going forward. Thus, as far as I'm concerned, my help to you to complete this work is actually just as important as improving documentation so, next time this comes up, somebody can just look at the docs.

Also, since we need to update genRubyInstaller.sh anyway, it's important to me that that is done properly so that, after this commit, repositioning changes in ruby.data go away. We need to eat that once (as part of this checkin), but we shouldn't need to eat that twice due to incorrect changes to genRubyInstaller.sh.

With the suggested changes to genRubyInstaller, future changes to Ruby will be much easier to understand from the file diff.

Thanks for your understanding. I consider this work mandatory for my signoff to this PR. You can get genRubyInstaller.sh (and documentation additions) checked in first, if you want, or you can include it in this PR. But I won't sign off on this one without seeing that first, and knowing that the messy ruby.data diffs are just a one-time thing.

Thanks for your understanding.